packages/create-sitecore-jss/src/init-runner.ts

```diff
@@ -35,12 +36,22 @@ export const initRunner = async (initializers: string[], args: BaseArgs) => {
   await runner(initializers);
 
   saveConfiguration(args.templates, path.resolve(`${args.destination}${sep}package.json`));
-
   // final steps (install, lint, etc)
   if (!args.noInstall) {
     installPackages(args.destination, args.silent);
     lintFix(args.destination, args.silent);
   }
+
+  // if you opt in to use the pre-commit hook for linting check,
+  // we need to initialize a git repository and install the hook locally.
+  if (args.preCommitHook === 'yes') {
+    exec(`cd ${args.destination} && git init && npm run install-pre-commit-hook`, (err: Error) => {
+      if (err) {
+        console.log('Error:', err);
```

Comment:
We might want to provide more context for this error in the message, and maybe de-escalate to a warning (not critical to the initializer overall). Something like:
```suggestion
        console.log(chalk.yellow(`Warning: Pre-commit hook may not be working due to error `${err}`));
  );
```